src/commands/version.js

```diff
@@ -0,0 +1,84 @@
+const dirsExist = require('../utils/dirs-exist');
+const { terminal } = require('terminal-kit');
+const path = require('path');
+const {
+  incrementPackageJsonVersion,
+  incrementVersionNumber,
+} = require('./../utils/increment-version');
+const { validateProject } = require('./../utils/projectpaths');
+const { getPackageVersion } = require('./../utils/get-package-version');
+const { findAllProjectPaths } = require('./../utils/get-version-project-paths');
```

Comment:
You don't need these custom functions, we already have prior art that covers both of these situations. As mentioned in [this comment](https://github.com/bigbite/build-tools/pull/119#discussion_r1657410841) you can use the same setup from the `build` command file.

If you look in the build command file, and at lines 62 through to 113 you will see the function in use that compiles the list of packages available to compile. There you will also find the code that extracts a object config which creates all variations of the paths required for you to use these, including a path to the `package.json` file and extracted version from that file.

To keep things consistent across both commands, use these and adapt them if you need.